File: Selenium/Project/CRM_Activity6.java
package Project;

import java.time.Duration;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class CRM_Activity6 {
	WebDriver driver;
	WebDriverWait wait;
    
    @BeforeClass
    public void setUp() {
        //Open a browser
        driver = new ChromeDriver();

       
        driver.get("http://alchemy.hguy.co/crm");
    }
    
    @Test(priority = 1)
	  public void loginFormTest() {
		  //Provide username and password
		  driver.findElement(By.id("user_name")).sendKeys("admin");
		  driver.findElement(By.id("username_password")).sendKeys("pa$$w0rd");
		  //Find the submit button and click it
		  driver.findElement(By.xpath("//input[@type='submit']")).click();
		  
		  
    }
    
    
    //Ensure that the “Activities” menu item exists
    
    @Test(priority = 2)
    public void locateMenu() throws InterruptedException {
    	Thread.sleep(5000);
    	driver.manage().window().maximize();
    	
    	WebElement activitiesMenu = driver.findElement(By.id("grouptab_3"));
    	String textAct = activitiesMenu.getText();
    	Thread.sleep(5000);
    	System.out.println("The Menu has " +  textAct);
    	Assert.assertTrue(activitiesMenu.isEnabled());
    	
    }

    @AfterClass
    public void tearDown() {
        driver.quit();
    }
    
}

File: Selenium/Project/CRM_Activity6.py
# Menu checking
# Goal: Make sure that the “Activities” menu item exists and is clickable

# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

# Start the Driver
with webdriver.Chrome() as driver:

    driver.get("https://alchemy.hguy.co/crm/")

# Find and select the username and password fields
# Enter login credentials into the respective fields
    driver.find_element(By.ID,"user_name").send_keys("admin")
    driver.find_element(By.ID,"username_password").send_keys("pa$$w0rd")
    driver.find_element(By.ID,"bigbutton").click()
    homePageHeader = driver.find_element(By.ID,"tab0").text
    # print(homePageHeader)
    if(homePageHeader=="SUITECRM DASHBOARD"):
        print("Home Page Opened Successfully !")
    else:
        print("Login failed !")

# Locate the navigation menu.
# Ensure that the “Activities” menu item exists
    driver.fullscreen_window()
    activitiesMenu = driver.find_element(By.ID, "grouptab_3")
    print("Activities Menu is visible? ", activitiesMenu.is_displayed())

# Close the browser
    driver.quit()